test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -104,7 +99,7 @@ describe( 'Demo content post', () => {
 		// Create the aligned right paragraph
 		await page.keyboard.press( 'Enter' );
 		await page.keyboard.type( '... like this one, which is right aligned.' );
-		await moveMouse(); // This shouldn't be necessary to show the toolbar
+		await page.keyboard.press( 'Escape' );
```

Comment:
My ideal solution would be to avoid this entirely and "show" the toolbar when `alt + f10` is hit. I don't know if that's possible though.

Reply:
I'd agree, and see this more as an interim step in the right direction (i.e. at least making it possible by keyboard-only). I'd put some thought toward what would be required to focus the non-visible contextual toolbar and I think it'll require some non-trivial state/lifecycle wizardry in BlockListBlock to have it be forced visible / focused.

Roughly, I imagine it looking like:

- `NavigableToolbar.KeybindScope` has a `onShortcutPressed` callback
- `BlockContextualToolbar` / `BlockToolbar` accepts an `autoFocus` prop which behaves similarly to the equivalently-named input attribute
- `BlockListBlock` assigns a callback for this which _somehow_ incurs a new render which triggers consideration in [`shouldShowContextualToolbar`](https://github.com/WordPress/gutenberg/blob/b6a56b2cd392e630b6d53a3be9d9a59d00c41d6f/packages/editor/src/components/block-list/block.js#L402) to force it to be shown, and _somehow_ assigns the `autoFocus` prop.

I'd like to reserve this for a subsequent pull request.